internal/validate/kube/utils.go

```diff
@@ -0,0 +1,49 @@
+package kube
```

Comment:
I would rename this file to `eks.go`. Using `utils` or `common` in go is considered a[ code smell ](https://dave.cheney.net/2019/01/08/avoid-package-names-like-base-util-or-common) (more so for package names) and these functions are used for EKS only anyhow.

Reply:
Done!